src/components/TopicSelector.less

```diff
@@ -20,11 +21,18 @@
 
   &__sort {
     color: #99aab5;
+    text-align: right;
+
+    span:first-of-type {
```

Comment:
Would it be better to use a className here rather than selecting the first span? Maybe use &__label for `className="TopicSelector__sort__label"` or `className="TopicSelector__sort__title"` or something else similar?